modules/checkout/tests/src/Functional/CheckoutOrderTest.php

```diff
@@ -155,13 +166,14 @@ public function testRegisterOrderCheckout() {
       'login[register][password][pass2]' => 'pass',
     ], 'Create account and continue');
     $this->assertSession()->pageTextContains('Billing information');
+    // Check breadcrumbs are not links. (the default setting)
+    $this->assertSession()->elementNotExists('css', '.block-commerce-checkout-progress li.checkout-progress--step > a');
```

Comment:
The test to use the link to move back and fwd through the checkout steps is in testGuestOrderCheckout. The test there is no link by default is in testRegisterOrderCheckout.